Documentation/batch-scripts/mentoring.py

```
@@ -0,0 +1,436 @@
+                cur_org = conn.cursor()
+                cur_org.execute(
+                    """SELECT organization_id, name, organization_code
+                       FROM organization_extension
+                       WHERE organization_id = %s""",
+                    (mentor_organization_id,)
+                )
+                org_row = cur_org.fetchone()
+                if org_row:
+                    org_id, org_name, org_code = org_row
+                cur_org.close()
+            except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)